algorithm/classification/naive_bayes.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Contoh implementasi klasifikasi menggunakan metode naive bayes
+ * @constant {object} class_ klasifikasi jumlah play yang bernilai yes dan no
+ * @function calc menghitung bobot klasifikasi yes dan no pada attribut tertentu
+ * @param {string} key setiap attribut pada record
+ * @param {string} value nilai attribut pada record
+ * @function naiveBayes memproses klasifikasi
+ * @param {object} data model testing
+ */
+
+const records = [
+    { outlook: 'Sunny', temperature: 'Hot', humidity: 'High', windy: false, play: 'no' },
+    { outlook: 'Sunny', temperature: 'Hot', humidity: 'High', windy: true, play: 'no' },
+    { outlook: 'Cloudy', temperature: 'Hot', humidity: 'High', windy: false, play: 'yes' },
+    { outlook: 'Rainy', temperature: 'Mild', humidity: 'High', windy: false, play: 'yes' },
+    { outlook: 'Rainy', temperature: 'Cool', humidity: 'Normal', windy: false, play: 'yes' },
+    { outlook: 'Rainy', temperature: 'Cool', humidity: 'Normal', windy: true, play: 'yes' },
+    { outlook: 'Cloudy', temperature: 'Cool', humidity: 'Normal', windy: true, play: 'yes' },
+    { outlook: 'Sunny', temperature: 'Mild', humidity: 'High', windy: false, play: 'no' },
+    { outlook: 'Sunny', temperature: 'Cool', humidity: 'Normal', windy: false, play: 'yes' },
+    { outlook: 'Rainy', temperature: 'Mild', humidity: 'Normal', windy: false, play: 'yes' },
+    { outlook: 'Sunny', temperature: 'Mild', humidity: 'Normal', windy: true, play: 'yes' },
+    { outlook: 'Cloudy', temperature: 'Mild', humidity: 'High', windy: true, play: 'yes' },
+    { outlook: 'Cloudy', temperature: 'Hot', humidity: 'Normal', windy: false, play: 'yes' },
+    { outlook: 'Rainy', temperature: 'Mild', humidity: 'High', windy: true, play: 'no' },
+]
+
+const class_ = { yes: 0, no: 0 }
+records.forEach((item) => {
+    item.play === 'yes' ? class_.yes++ : class_.no++;
```

Comment:
Bagian kudu make if-else dibanding shorthand begini, soalnya ga rapih dan dianggap error sama [codacy](https://app.codacy.com/gh/bellshade/Javascript/pullRequest?prid=9596149).

```suggestion
    if (item.play === "yes") {
        class_.yes++;
    } else {
        class_.no++;
    }
```